src/streamlink/plugins/ert.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+
+import re
+import requests
+
+from streamlink.plugin import Plugin
+from streamlink.plugin.api import http, validate, useragents
+
+YOUTUBE_URL = "https://www.youtube.com/watch?v={0}"
+req = requests.get('http://extreme-ip-lookup.com/json/', params={'User-Agent': useragents.CHROME})
+j_son = req.json()
+
+_youtube_id = re.compile(r'/embed/([\w-]+)(?:.+?/embed/([\w-]+)|)', re.M | re.S)
+
+_url_re = re.compile(r'http(s)?://webtv\.ert\.gr/ert.+?/')
+
+
+class Ert(Plugin):
+
+    @classmethod
+    def can_handle_url(cls, url):
+
+        return _url_re.match(url)
+
+    def _yt_id(self):
+
+        r1 = requests.get(self.url, params={'User-Agent': useragents.CHROME})
+
+        iframe_url = re.search(r'iframe src="(.+?)"', r1.content, re.U).group(1)
```

Comment:
There is a method `streamlink.plugin.api.utils.itertags` that is more robust than this regex, you might find it nicer to use that.